about.html

```diff
@@ -0,0 +1,214 @@
+                After online public voting and Selection Communities judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimised the prize-winning-logo for this year's summit as following:
+            </p>
+
+            <div class="logo_container">
+
+                <div class="log2_cont">
+
+                    <div class="log2">
+                        <img class="log2_img" src="images/logo.png" alt="conference logo">
+                        <div class="log2_title">
+                            <div class="log2_title1">CM</div>
+                            <div class="log2_title2">Summit 2015</div>
+                        </div>
+                    </div>
+
+                    <div class="logo_date">14th-17th Oct | K'la, Uganda</div>
+
+                </div>          
+                
+            </div>
+        </div>
+
+        <hr class="logo_divider">
+        <section class="past_summit">
+            <h3 class="summit_head">
+                See the past CM International Summits
+            </h3>
+            <hr class="about_them">
+            <p class="summit_para">
+                Take a look at the las two Kampala Programmers Forums which took place in Buenos Aire and Warsaw.
+            </p>
+
+            <div class="summit_img">
+                <div class="summit_img_one">
+                    <div class="year"><strong>2013</strong></div> <br>Kampala Programmers Forum 2013 in Buenos Aires
+                </div>
+    
+    
+                <div class="summit_img_two">
+                    <div class="year"><strong>2011</strong></div> <br>Kampala Programmers Forum 2013 in Warsaw
+                </div>
+
+            </div>           
+
+        </section>
+
+        <section class="partners">
+            <h2 class="partner_head">Partner</h2>
+
+            <hr class="about_them">
+            <div class="partner_container">
+
+                <img class="partner_img" src="images/partners/mozilla.png" alt="mozilla">
+
+                <img class="partner_img" src="images/partners/netflix.png" alt="Netflix">
+
+                <img class="partner_img" src="images/partners/amazon.png" alt="Amazon">
+
+
+                <img class="partner_img" src="images/partners/ibm.png" alt="IBM">
+
+                <img class="partner_img" src="images/partners/airbnb.png" alt="airbnb">
+
+            </div>
+
+        </section>
+
+        <footer class="footer">
+
+                <div class="right_info">
+                    <div class="small_trade">CM</div>
+                    <div class="trade_two">Coding Minds <br>
+                        International Summit
+                    </div>
+                </div>            
+           
+            <div class="left_info">
+                2015 Coding Minds International Summit
+                <br>Some rights reserved
+            </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly `About` page's footer colors different than `Home` page but you use same design both pages therfore please fix it  and check again [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/219433913-bb6e0d89-a632-4967-aa8e-48e4294f7df8.png)